dd-sdk-android/src/main/kotlin/com/datadog/android/webview/internal/log/WebLogEventDateCorrector.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.webview.internal.log
+
+import com.datadog.android.core.internal.time.TimeProvider
+import com.datadog.android.core.internal.utils.sdkLogger
+import com.datadog.android.log.internal.utils.buildSimpleDateFormat
+import java.lang.IllegalArgumentException
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+internal class WebLogEventDateCorrector(
+    private val timeProvider: TimeProvider,
+    private val simpleDateFormat: SimpleDateFormat =
+        buildSimpleDateFormat()
+) {
+
+    fun correctDate(date: String): String? {
+        return correctLogDate(date)
+    }
+
+    private fun correctLogDate(date: String): String? {
+        synchronized(simpleDateFormat) {
+            return try {
+                val time = simpleDateFormat.parse(date)?.time
```

Comment:
so for RUM events `browser` sends `date` field as timestamp, but for Log events it is string in ISO format? Do you know why there is such difference?

Reply:
I think this is mostly that Logs backend API was done long time ago and that was the format expected. We had to follow what was there.